types/wlr_xdg_shell_v6.c

```diff
@@ -18,22 +18,9 @@ static const char *wlr_desktop_xdg_popup_role = "xdg_popup_v6";
 
 struct wlr_xdg_positioner_v6 {
 	struct wl_resource *resource;
-
-	struct wlr_box anchor_rect;
-	enum zxdg_positioner_v6_anchor anchor;
-	enum zxdg_positioner_v6_gravity gravity;
-	enum zxdg_positioner_v6_constraint_adjustment constraint_adjustment;
-
-	struct {
-		int32_t width, height;
-	} size;
-
-	struct {
-		int32_t x, y;
-	} offset;
+	struct wlr_xdg_positioner_v6_attributes *attrs;
```

Comment:
Is there a reason to have this a pointer?
The wlr_xdg_popup_v6 has it inline, and it's allocated and free()d together with the positioner struct

Reply:
bump